tests/netebpfext_unit/netebpfext_unit.cpp

```diff
@@ -659,6 +673,14 @@ sock_addr_thread_function(
         }
 
         auto expected_result = _get_fwp_sock_addr_action(port_number);
+
+        // SOCK_ADDR_TEST_ACTION_PERMIT_HARD currently isn't supported in receive
+        // Workaround for now - map to block
```

Comment:
```suggestion
        // SOCK_ADDR_TEST_ACTION_PERMIT_HARD currently isn't supported in receive.
        // Workaround for now - map to block.
```
Style convention in this file is to end comments with punctuation.